bindings/spt/sys_linux_ppc64le.c

```diff
@@ -42,13 +42,17 @@ long sys_read(long fd, void *buf, long size)
     register long r3 __asm__("r3") = fd;
     register long r4 __asm__("r4") = (long)buf;
     register long r5 __asm__("r5") = size;
+    long cr;
 
     __asm__ __volatile__ (
-            "sc"
-            : "=r" (r3)
+            "sc\n\t"
```

Comment:
Nit: Superfluous `\n\t` here and in other occurrences of `sc`?

Reply:
we need either `\n\t` or a ';' after the `sc`, otherwise we get:
```
CC spt/sys_linux_ppc64le.c
/tmp/ccDPEtcA.s: Assembler messages:
/tmp/ccDPEtcA.s:21: Error: unrecognized opcode: `scmfcr'
GNUmakefile:67: recipe for target 'spt/sys_linux_ppc64le.o' failed
make[1]: *** [spt/sys_linux_ppc64le.o] Error 1
GNUmakefile:69: recipe for target 'bindings' failed
```

Fixed a type in the header file, though...